components/cart.jsx

```diff
@@ -108,7 +111,14 @@ export const CartSidebar = () => {
       return;
     }
 
-    setCheckoutPayload({ variables: getCheckoutPayload(items) });
+    client
```

Comment:
Pretty sure the Apollo client returns a promise. You could await `client.mutate` since this is an async fn. Or you could de-async it and use this construction. 

Strictly speaking though, you should handle this async pattern via the `useEffect` hook and `useMutation` the way the Stripe starter did. Fire-and-forget functions like this lead to memory leaks.